src/sentry/monitors/tasks.py

```diff
@@ -425,6 +425,9 @@ def detect_broken_monitor_envs():
     open_incidents_qs = MonitorIncident.objects.select_related("monitor").filter(
         resolving_checkin=None,
         starting_timestamp__lte=(current_time - timedelta(days=NUM_DAYS_BROKEN_PERIOD)),
+        monitor__status=ObjectStatus.ACTIVE,
+        monitor__is_muted=False,
+        monitor_environment__is_muted=False,
```

Comment:
Hmm, if a monitor is muted we probably still want to stop spamming missed checkins though... Not sure if we should actually exclude muted monitors here?

Reply:
ooh muted monitors don't create missed checkins to my knowledge right now

Reply:
Ahh you're right, nice! https://github.com/getsentry/sentry/blob/f968500ac3e770c04304c594ee7a7ce164ba5a76/src/sentry/monitors/tasks.py#L268-L271